CONTRIBUTING.md

```diff
@@ -0,0 +1,318 @@
+
+# Developer guide
+
+## Introduction
+
+The Scap Security Guide (SSG) consists of a collection of SCAP content for different platforms such as Red Hat Enterprise Linux, Fedora, Debian, Suse and others.
+It is created and maintained by the community, and anyone can join this effort by creating new or improving the current content.
+There are three main types of content in SSG, they are rules, defined using the XCCDF standard, checks, usually written in [OVAL](https://oval.mitre.org/language/about/) format, and remediations, that can be executed on ansible, bash, anaconda installer and puppet.
+SSG also has its own templating mechanism, allowing content writers to create models and use it to generate a number of checks and remediations.
+
+The SSG content is primarily divided by platform and it can be seen on its directory structure:
+
+> **scap-security-guide/**
+├── Chromium
```

Comment:
build?

Reply:
Indeed, it seems GitHub sorting algorithm takes the case into account. I looked for **build** directory before the **Chromium** directory and it was not there.